app/assets/javascripts/questionnaire_logic.js.erb

```diff
@@ -37,6 +37,26 @@ function toggle_shown_questions() {
   });
 }
 
+function toggle_hidden_questions() {
+  if ($(this).closest('.row').find('input[data-hides-questions]').length === 0) return;
+  $(this).closest('.row').find('input[data-hides-questions]').each(function (index) {
+    if ($(this).is(':checked')) {
```

Comment:
je kunt die if ook binnen de `forEach` brengen, dan heb je nog maar 1 foreach nodig. Daar wordt het denk ik wel duidelijker van. Let wel op dat je dan nog een goeie reference naar `$(this)` hebt.

Reply:
En check ook even hoe het zit met die functie die hierboven staat. Die `if` met toggle kun je in een functie zetten en dan ook gebruiken in de andere functies waar dingen getoggled moeten worden. Daarnaast kun je misschien nog wel meer delen hergebruiken. In principe doet het ongeveer het zelfde toch?

Reply:
Ja goeie. Ik heb het aangepast.